sys/select_32.go

```diff
@@ -1,4 +1,4 @@
-// +build darwin dragonfly freebsd netbsd openbsd
+// +build darwin 386,freebsd arm,freebsd 386,linux arm,linux netbsd openbsd
```

Comment:
You need both select_{bsd,linux}.go **and** select_{32,64}.go. The first pair is due to the fact that Select has different signatures on BSD and Linux.